spec/item_repository.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../lib/item'
+
+class ItemRepository
+    def all
+        items = []
+
+        sql = 'SELECT * FROM items;'
+        result_set = DatabaseConnection.exec_params(sql, [])
+        result_set.each do |record|
+            new_item = Item.new
+            new_item.id = record['id']
+            new_item.quantity= record['quantity']
+            new_item.name = record['name']
+            new_item.unit_price = record['unit_price']
```

Comment:
In the Item class I would put all of these in a constructor so whenever I make a new item I don't have to dedicate this many lines of code. So when calling Item.new I can pass in (record['id'], record['quantity']... etc).